packages/subgraph/src/controller.ts

```diff
@@ -167,12 +169,20 @@ export function handleDepositCollateral(event: DepositCollateral): void {
     dayStatSnapshot.totalCollateralAmount.plus(event.params.amount);
   dayStatSnapshot.save();
 
-  const transactionHistory = createTransactionHistory("DEPOSIT_COLLAT", event);
+  let transactionHistory = createTransactionHistory("DEPOSIT_COLLAT", event);
   transactionHistory.owner = vault.owner;
   transactionHistory.ethAmount = event.params.amount;
   transactionHistory.save();
 
-  const position = loadOrCreatePosition("SHORT", vault.owner);
+  let usdcPrices = getETHUSDCPrice();
+  let amount = BigDecimal.fromString(event.params.amount.toString());
+
+  let position = loadOrCreatePosition(vault.owner, "SHORT");
```

Comment:
Should be positionType followed by owner addr?
`const position = loadOrCreatePosition("SHORT", vault.owner)`

Reply:
@adam5909099